tests/meta/upload/vogue/conftest.py

```diff
@@ -8,24 +8,24 @@
 
 
 GTCONFIG = {
-    'genotype': {
-        'database': 'database',
-        'config_path': '/Users/mayabrandi/opt/config/genotype.yaml',
-        'binary_path': '/Users/mayabrandi/opt/cg/genotype_path'
-        }
-    }
-VOGUECONFIG = {
-    'vogue': {
-        'binary_path': '/Users/mayabrandi/opt/cg/vogue_path'
-        }
+    "genotype": {
+        "database": "database",
+        "config_path": "/Users/mayabrandi/opt/config/genotype.yaml",
```

Comment:
It looks like you might be using the real deal, I suggest we don't if that is the case
```suggestion
        "config_path": "/path/to/config/genotype.yaml",
```